packages/core/upload/admin/src/pages/App/components/BulkActions.tsx

```diff
@@ -29,18 +34,15 @@ export const BulkActions = ({ selected, onSuccess, currentFolder }) => {
         )}
       </Typography>
 
-      <BulkDeleteButton selected={selected} onSuccess={onSuccess} />
-      <BulkMoveButton currentFolder={currentFolder} selected={selected} onSuccess={onSuccess} />
+      <BulkDeleteButton
+        selected={selected as BulkDeleteButtonProps['selected']}
```

Comment:
In general there is a lot of type casting going on when passing props. I don't think it's a good sign. We should be passing values the component expects. Could you take a look at the places where you are typecasting props and see if it can be fixed by improving the types?